importer/tests/writer_test.py

```diff
@@ -27,9 +34,67 @@ def test_copy_files_single_strand(self, copyfile_patch):
             RawRead(forward_read='PAIR1_1.fastq.gz', reverse_read='PAIR1_2.fastq.gz', sample_name='SAMPLE1',
                     taxon_id='1280', library_name='LIB1', sample_accession=None),
             RawRead(forward_read='SINGLE.fastq.gz', reverse_read=None, sample_name='SAMPLE2',
-                    taxon_id='1280', library_name='LIB2', sample_accession=None)]), 'destination', 0)
+                    taxon_id='1280', library_name='LIB2', sample_accession=None)]), 'destination', 0, 0)
         under_test.copy_files('source')
         self.assertEquals(copyfile_patch.call_args_list,
                           [call('source/PAIR1_1.fastq.gz', 'destination/0/PAIR1_1.fastq.gz'),
                            call('source/PAIR1_2.fastq.gz', 'destination/0/PAIR1_2.fastq.gz'),
                            call('source/SINGLE.fastq.gz', 'destination/0/SINGLE.fastq.gz')])
+
+
+class TestXlsGeneration(unittest.TestCase):
+
+    def test_of_OutputSpreadsheetGenerator(self):
+        sheet = self.make_spreadsheet_for_test()
+        self.run_tested_function(sheet)
+        self.testing_loop()
+        os.remove('workbook.xls')
+
+    def testing_loop(self):
```

Comment:
I think a name like "assert_generated_workbook" is a lot clearer on the intention
(This method is not "testing a loop" if that makes sense)